app/src/test/java/com/example/bottomnav/ParseTest.java

```diff
@@ -174,4 +173,26 @@ public void parseData() throws Exception {
         assertEquals("uint8_t", struct.get(4).type);
         assertEquals("uint8_t", struct.get(5).type);
     }
+
+    @Test
+    public void checkIDToStructMap() throws Exception {
+        Parse test = Parse.parseTextFile("parseData.h");
+
+        //Given CAN ID name, retrieve its associated structure contents
+        ArrayList<StructContents> struct1 = test.getCanStruct("CAN_CHARGER_STATUS");
+        ArrayList<StructContents> struct2 = test.getCanStruct("CAN_CHARGER_CONTROL");
+        ArrayList<StructContents> struct3 = test.getCanStruct("CAN_STRAIN_DATA");
+        ArrayList<StructContents> struct4 = test.getCanStruct("CAN_PEDAL_POS");
+
+        //Retrieve contents of structure from PayLoadMap object
+        ArrayList<StructContents> stru1 = test.getStruct("ChargerStatusStruct");
+        ArrayList<StructContents> stru2 = test.getStruct("ChargerControlStruct");
+        ArrayList<StructContents> stru3 = test.getStruct("CanStrainGaugeStruct");
+        ArrayList<StructContents> stru4 = test.getStruct("CanPedalPosStruct");
+
+        assertEquals(stru1, struct1);
```

Comment:
Can you inline these? `struct1` and `stru1` don't really add anything in terms of readability (and similarly for below)
```suggestion
        assertEquals(test.getStruct("ChargerStatusStruct"), test.getCanStruct("CAN_CHARGER_STATUS"));
```